### PR TITLE
feat(Recipes): New Recipe ent-mysql

### PR DESCRIPTION
**Pull request by Eeap**
Hello, maintainers ! 
This is recipe to use ent ORM for Fiber with MySQL. I hope this recipe is useful 😊

### CODE REVIEW

File: ent-mysql/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+### Example ent ORM for fiber with MySQL
```
Comment (gaby) [inline ent-mysql/README.md:1]: Can you please add the recipe to `.github/dependabot.yml` file

File: ent-mysql/go.mod [hunk 2]
```diff
@@ -0,0 +1,37 @@
+module http
+
+go 1.20
```
Comment (gaby) [inline ent-mysql/go.mod:3]: this can probably be `1.19`

File: ent-mysql/go.mod [hunk 3]
```diff
@@ -0,0 +1,37 @@
+module http
```
Comment (gaby) [inline ent-mysql/go.mod:1]: why not `ent-mysql` ?

**Review by gaby — CHANGES_REQUESTED** [review]
Run the whole recipe through this:

```
gofumpt -l -w .
```

**Comment by gaby** [comment]
@Eeap  Thanks for the submissions, just a few small changes :-) May also want to list the new recipe in the main README.md file

**Comment by Eeap** [comment]
> @Eeap Thanks for the submissions, just a few small changes :-) May also want to list the new recipe in the main README.md file

Okay, Thank you for review. I have changed it

**Review by gaby — APPROVED** [review]

File: ent-mysql/routes/routes.go [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+	"github.com/gofiber/fiber/v2"
+)
+
+var ctx = context.Background()
+
+func GetBook(c *fiber.Ctx) error {
+	id, _ := strconv.Atoi(c.Params("id"))
+	b, err := database.DBConn.Book.
+		Get(ctx, id)
+	if err != nil {
```
Comment (efectn) [inline ent-mysql/routes/routes.go:18]: We should check not found error here and return 404

Comment (Eeap) [reply ent-mysql/routes/routes.go:21]: > We should check not found error here and return 404

Oh, I see. I just added the error handling ! :)

**Comment by efectn** [comment]
Why don't you check Post quieries when creating or updating? It should be better than hardcoded valuea

**Comment by Eeap** [comment]
> Why don't you check Post quieries when creating or updating? It should be better than hardcoded values

I changed hardcoded values into post queries data and added empty check(if empty, return 400)

**Review by efectn — APPROVED** [review]

**Comment by gaby** [comment]
@ReneWerner87 this one is also done, although now it has a conflict 😂

**Comment by gaby** [comment]
@Eeap can take a look at the conflict, it's related to your other PR getting merged :-)

**Comment by Eeap** [comment]
@gaby I checked that. Thanks for letting me know that conflict 😊

**Pull request merged**